src/main/java/homework/task1/Task1.java

```diff
@@ -0,0 +1,20 @@
+package homework.task1;
+
+public class Task1 {
+    public static void main(String[] args){
+
+        if(args.length > 0) {
+            for (int i = 0; i < args.length; i++) {
+
+                String toPrint = args[i];
+                if (toPrint.matches("error")){                                //checking for error
+                    System.out.println("____Alarm!____");
+                    System.exit(0);
```

Comment:
It's not recommended way to exit a program

Reply:
removed